docs/howto/write-logs.rst

```diff
@@ -115,6 +115,29 @@ To configure it, you must additionally set the endpoint url to point to your loc
 You can do this via the Connection Extra ``host`` field.
 For example, ``{"host": "http://localstack:4572"}``
 
+.. _write-logs-amazon-cloudwatch:
+
+Writing Logs to Amazon Cloudwatch
+---------------------------------
+
+
+Enabling remote logging
+'''''''''''''''''''''''
+
+To enable this feature, ``airflow.cfg`` must be configured as follows:
+
+.. code-block:: ini
+
+    [logging]
+    # Airflow can store logs remotely in AWS Cloudwatch. Users must supply a log group
+    # ARN (starting with 'cloudwatch://...') and an Airflow connection
+    # id that provides write and read access to the log location.
+    remote_logging = True
+    remote_base_log_folder = cloudwatch://arn:aws:logs:<region name>:<account id>:log-group:<group name>:*
```

Comment:
What is the `*` on the end here for? What other possible values would it have?

Reply:
When you display the ARNs in the cloudwatch console or via a describe-log-groups call, they all have a '*' suffix.

The docs however describe the arn without a suffix: https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazoncloudwatchlogs.html#amazoncloudwatchlogs-resources-for-iam-policies

I'm happy to go with whichever format you think is less confusing.